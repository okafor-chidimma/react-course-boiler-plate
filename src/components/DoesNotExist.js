import React from 'react';
import { Link } from 'react-router-dom';

const DoesNotExist = () => (
  <div>
    404page!<Link to="/">Home</Link>
  </div>
);

export default DoesNotExist;