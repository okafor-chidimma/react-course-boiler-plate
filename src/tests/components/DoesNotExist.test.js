import React from 'react';
import { shallow } from 'enzyme';
import DoesNotExist from '../../components/DoesNotExist';

describe('Rendering <DoesNotExist/>', () => {
  it('Renders <DoesNotExist/> correctly', () => {
    const wrapper = shallow(<DoesNotExist />);
    expect(wrapper).toMatchSnapshot();
  })
})