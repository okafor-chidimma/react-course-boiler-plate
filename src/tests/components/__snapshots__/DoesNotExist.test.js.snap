// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering <DoesNotExist/> Renders <DoesNotExist/> correctly 1`] = `
<div>
  404page!
  <Link
    to="/"
  >
    Home
  </Link>
</div>
`;
